tsl/test/sql/dist_policy.sql

```diff
@@ -0,0 +1,59 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+
+\unset ECHO
+\o /dev/null
+\ir include/remote_exec.sql
+\o
+\set ECHO all
+
+CREATE OR REPLACE FUNCTION test_drop_chunks(job_id INTEGER)
```

Comment:
This name is confusing, since policy was renamed and it tests the retention policy:
```suggestion
CREATE OR REPLACE FUNCTION test_retention(job_id INTEGER)
```